### PR TITLE
[14.0][MIG] hr_attendance_report_theoretical_time

### PR DESCRIPTION
**Pull request by ao-landoo**
Standard migration to 14.0

### CODE REVIEW

**Comment by ao-landoo** [comment]
@pedrobaeza I know there is another PR but looks like author is not responding. He has many PRs but there is no answer in any of them. Can I get workflow approval?

**Review by ghost — APPROVED** [review]

**Review by dsolanki-initos — APPROVED** [review]
LGTM. Code review.

File: hr_attendance_report_theoretical_time/views/hr_employee_views.xml
Comment (oihane) [inline hr_attendance_report_theoretical_time/views/hr_employee_views.xml]: Runbot is red because of this definition:

```
2021-07-02 09:08:08,833 194 WARNING openerp_test py.warnings:
/home/odoo/odoo-14.0/odoo/tools/convert.py:336: 
DeprecationWarning: The <act_window> tag is deprecated,
use a <record> for 'recompute_employee_theoretical_attendance'.
warnings.warn(f"The <act_window> tag is deprecated,
use a <record> for {xml_id!r}.", DeprecationWarning)
```
Can you check it please.

**Comment by dsolanki-initos** [comment]
Hi @ao-landoo 
it would be great if you can check above comments

**Comment by ao-landoo** [comment]
@dsolanki-initos runbot is green

**Review by dsolanki-initos — APPROVED** [review]
LGTM

**Comment by ao-landoo** [comment]
Can this get merged?

**Comment by ao-landoo** [comment]
@pedrobaeza @dreispt Could you merge this PR?

**Comment by pedrobaeza** [comment]
Please squash administration commits with their real ones. Example provided, but there can be more:

Selección_054

**Comment by ao-landoo** [comment]
@pedrobaeza Squashed administrative commits related to V13. I would say didn't mess anything up, it would still be nice if you could check.

**Comment by ao-landoo** [comment]
@pedrobaeza Is it normal that OCA bot is taking almost a day merging to branch 14.0?

**Comment by pedrobaeza** [comment]
Please rebase over latest origin to see the problem.

**Comment by ao-landoo** [comment]
@pedrobaeza Rebased, let's see now.

**Comment by pedrobaeza** [comment]
@sbidoul it seems something is misconfigured in MAKEPOT part:

https://app.travis-ci.com/github/OCA/hr-attendance/jobs/532270141#L1271

Maybe the token?

**Comment by sbidoul** [comment]
`Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.`... that is not going to be fun. Let me see.

**Comment by sbidoul** [comment]
@pedrobaeza can you retry the merge. Hopefully  OCA/maintainer-quality-tools#694 is enough to resolve this.

**Comment by pedrobaeza** [comment]
Thanks for solving it, Stéphane

**Pull request merged**
